### DISCUSSION TITLE
how the .csv files are generated?

### DISCUSSION THREAD

**Discussion by elenitskorda** [Nesvilab/FragPipe-Analyst-doc#12] [Q&A] [answered]
Hi,

I am a master student working on ovarian cancer. Fragpipe is part of my thesis and a very helpful tool! Thank you for your work.

My question is when for example when your experiment type is TMT, and you upload the necessaty .tsv files in order to create the report, how are the datasets of DE results.csv, Original matrix.csv etc are generated? Do you have the R code for that somewhere? 

It will be very helpful :)

Best, 
Eleni

<img width="771" alt="dataset" src="https://github.com/Nesvilab/FragPipe-Analyst/assets/32453911/17a6ae13-144c-4784-855f-0fc57d95a3a8">

**Comment by hsiaoyi0504** [comment, marked as answer]
@elenitskorda Sorry for the late reply. The source code of FragPipe-Analyst is here: https://github.com/MonashProteomics/FragPipe-Analyst. Based on the experiment type and the imputation option used, but as default for TMT, non-imputed data will be subjected to differential expression analysis between conditions using Limma.
